tests/combined_energy_corrected_eff_at_quad_trap_frequencies.json

```diff
@@ -0,0 +1 @@
716030107938787, 0.021873575568506547, 0.02213376711141553, 0.02180967105578097, 0.021825906254542813, 0.021110848836749463, 0.020076638321507564]], "eff interp with slope correction": [1.4668660538623184, 1.4164193230411954, 1.3721942122115842, 1.3298924400354186, 1.2809448274135948, 1.256160461603891, 1.249027430428252, 1.2355207574551526, 1.2480308723897449, 1.2513926404413505, 1.2836474876419066, 1.232157477761166, 1.2464145538085383, 1.234143420917021, 1.2297731169067514, 1.2204931907750967, 1.1675379547595301, 1.1391669203820984, 1.1484380386094646, 1.1275644395517637, 1.125216817498482, 1.0898615546274018, 1.09825819913958, 1.0867973745308057, 1.0718573000027698, 1.061298742500352, 1.0675627305873734, 1.0418572148125809, 1.0460062589480068, 0.9982345799935061, 0.9964960472314726, 0.9821161615871357, 0.9655974432563255, 0.9677478204953629, 0.9211458703567973, 0.9054527195655039, 0.871717841334733, 0.8423691706108363, 0.7231856092247476, 0.7065073701336197, 0.7210992891587926, 0.7365696693094375, 0.762223626129196, 0.7378582422036585, 0.7122631386330306, 0.6976571881678618, 0.7025207537752179, 0.6650311915107378, 0.6707344600569014, 0.6421963297988762, 0.6290604054112272, 0.607971695869184, 0.5827895115957884, 0.5626988283536013, 0.558610574038487, 0.5334987293156338, 0.5153064635696207, 0.5181163803372295, 0.5031313680348068, 0.4931261476587655, 0.4684690773470708, 0.4532191358123547, 0.4330940044384025, 0.40969369201162836, 0.4009610308194194, 0.374869310633167, 0.356654237907291, 0.3448406092477334, 0.3326119210427899, 0.30000383435039385, 0.2928433503550814], "frequencies": [25848704944.874474, 25850660208.08759, 25854656919.854702, 25856713444.67312, 25858654731.075645, 25860721662.890625, 25862690378.62796, 25864764757.799557, 25866680120.76201, 25868655112.681347, 25870630013.02136, 25872613707.6673, 25874641617.057472, 25876680285.819027, 25878561196.638554, 25880569320.1901, 25882782782.313133, 25884635784.92305, 25886702201.63384, 25888666230.560036, 25890550000.899635, 25892673717.955124, 25894553483.60415, 25896619208.967377, 25898618301.15566, 25900580999.32251, 25902649445.79724, 25904603312.470394, 25906600310.623363, 25908412871.151928, 25908580596.98165, 25910543276.471016, 25912594665.335064, 25914516985.474037, 25916568512.617626, 25918457740.680573, 25920479743.17804, 25922450859.671406, 25924392020.29544, 25926593005.844406, 25928728022.234745, 25930593981.300648, 25932691104.998096, 25934531337.73365, 25936543192.668655, 25938663489.72757, 25940453986.839478, 25942478583.29003, 25944407982.82288, 25946541271.34778, 25948474757.479637, 25950328317.98509, 25952428362.35367, 25954560787.596706, 25956546400.184437, 25959121639.90925, 25960377976.311066, 25962466483.26102, 25964852403.714535, 25966550076.102276, 25968411840.695835, 25970469381.924213, 25972365023.335304, 25974318811.47283, 25976332025.05481, 25978497091.76154, 25980445598.397015, 25982381957.574455, 25985130905.80705, 25987129137.957024, 25988455015.161922]}
```

Comment:
are these actual results or inputs for the analysis?
if so, move them to e.g. termite and used a much simpler example for this test.
else ignore this comment!